homeassistant/components/light/lcn.py

```diff
@@ -0,0 +1,118 @@
+"""
+Support for LCN lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.lcn/
+"""
+
+import voluptuous as vol
+
+from homeassistant.components.lcn import (
+    CONF_DIMMABLE, CONF_OUTPUT, CONF_TRANSITION, OUTPUT_PORTS, LcnDevice,
+    is_address)
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_TRANSITION, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS,
+    SUPPORT_TRANSITION, Light)
+from homeassistant.const import CONF_ADDRESS, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = ['lcn']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
Move the device config to the component so that all config is under the component. See eg the rainmachine component.

Load the platform via our discovery helper.

Reply:
Nice! Didn't even know this is possible. I like that.